chess-1/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "chess-1",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.0",
+    "@testing-library/user-event": "^7.2.1",
+    "eslint": "^6.8.0",
+    "husky": "^4.2.0",
+    "lint-staged": "^10.0.2",
+    "prettier": "^1.19.1",
+    "prettier-standard": "^16.1.0",
```

Comment:
This need to be moved to dev dependencies

Reply:
Moved.